app/assets/javascripts/admin/addon/templates/components/staff-actions.hbs

```diff
@@ -0,0 +1,4 @@
+{{!-- template-lint-disable no-invalid-interactive --}}
```

Comment:
I would prefer not having this, but given we use event delegation here, I cant say this div has the role of a button.